jina/parsers/peapods/pod.py

```diff
@@ -142,3 +142,12 @@ def mixin_k8s_pod_parser(parser):
         if _SHOW_ALL_ARGS
         else argparse.SUPPRESS,
     )
+    parser.add_argument(
+        '--k8s-custom-resource-dir',
+        type=str,
```

Comment:
set default value to None

Reply:
Done in https://github.com/jina-ai/jina/pull/3309/commits/6f60fe45069236ba73d80bd5d07c514cf99dd229